src/ensembl/src/content/app/species/state/sidebar/speciesSidebarSlice.ts

```diff
@@ -68,7 +72,8 @@ type SpeciesPageSidebarState = {
 };
 
 const initialState: SpeciesPageSidebarState = {
-  isOpen: true,
+  isOpen:
+    speciesStorageService.getSidebarState()?.isOpen === false ? false : true,
```

Comment:
`speciesStorageService.getSidebarState()?.isOpen ?? true`

Reply:
Cool! Sure IE won't cry?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Or I guess it is translated by Babel to support IE?

Reply:
We both use babel transform AND do not support IE 😄

Reply:
Ah yes! even better 👍